Utilities/XrdAdaptor/src/XrdRequestManager.cc

```diff
@@ -331,13 +397,27 @@ bool RequestManager::compareSources(const timespec &now,
     return false;
   }
 
+  std::string hostname_a;
+  std::string hostname_b;
```

Comment:
Could you move these declarations to the innermost scope where they are needed`(i..e right before the corresponding `Source::getHostName()` call)?